pipelines/imputation/scientificValidation/README.md

```diff
@@ -59,3 +59,42 @@ we usually use `gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_asse
 #### Outputs
 * output_vcf - reheadered vcf file
 * output_vcf_index
+
+## BeagleValidation
+### Docker
+Be sure to update the jars in `beagle_jars` if you
+want to create a new image with updated jars
+
+When building the docker image and you are on a M1 mac machine
```

Comment:
could you add a `docker_build.sh` script to make it super easy/straightforward to build new images? something like this (can probably cut a bunch out of this, doesn't need all these bells and whistles) https://github.com/broadinstitute/warp-tools/blob/develop/3rd-party-tools/snapatac2/docker_build.sh

Reply:
what if i add the docker build command to run to build it.  I don think we'll even be building this image again and even if we did it would just be running one command is all.  Its pretty straight forward for this kind of image.